client/src/router.tsx

```diff
@@ -13,13 +12,11 @@ export default function Router() {
   return (
     <Routes>
       <Route path="/" element={<HomePage />} />
-      <Route element={<Layout />}>
-        <Route path="/atlas/*" element={<AtlasRoutes />} />
-        <Route path="/european-projects/*" element={<EuropeanProjectsRoutes />} />
-        <Route path="/open-alex/*" element={<OpenAlexRoutes />} />
-        <Route path="/teds/*" element={<TedsRoutes />} />
-        <Route path="/finance-universite/*" element={< FinanceUniversityRoutes />} />
-      </Route>
+      <Route path="/atlas/*" element={<AtlasRoutes />} />
```

Comment:
I do not get the reason why you moved the Layout element from here to each page ?

Reply:
each sub site has to manage his translation